### DISCUSSION TITLE
Welcome to social-media-buttons-toolbar Discussions!

### DISCUSSION THREAD

**Discussion by ArthurGareginyan** [ArthurGareginyan/social-media-buttons-toolbar#6] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kaputze** [comment]
It would be nice to have an option for XMPP/Jabber. And: thanks for your work!
